R/Rules-class.R

```diff
@@ -35,7 +35,8 @@ setClass(
 #'
 #' [`NextBestMTD`] is the class for next best dose based on MTD estimate.
 #'
-#' @slot target (`proportion`)\cr target toxicity probability.
+#' @slot target (`proportion`)\cr target toxicity probability, except \eqn{0} or
+#'   \eqn{1}.
```

Comment:
for maintenance purpose, highly recommend to just say 1 or 0 without the markup around it.

Reply:
ohhh, sure, then It would be a lot o changes now :)

Reply:
you can try find and replace across files, tip:

Replace a string with another one in multiple files in this directory:
`perl -pi -w -e 's/OldText/NewText/g;' *.txt`